app/test_main.py

```
@@ -1,3 +1,64 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years,dog_years", [
+        (0, 0),
+        (14, 14),
+        (15, 15),
+        (23, 23),
+        (27, 27),
+        (-34, -40),
```

The task description does not specify how to handle negative input values. If negative values are not allowed, the test should either be removed or the `get_human_age` function should raise an error for negative inputs. If negative values are acceptable, the `get_human_age` function should handle them appropriately according to the problem description.